nifi-frontend/src/main/frontend/apps/standard-content-viewer/src/app/pages/standard-content-viewer/feature/standard-content-viewer.component.html

```diff
@@ -33,7 +33,7 @@ <h3 class="primary-color">
             @if (error) {
                 <div>{{ error }}</div>
             } @else {
-                <div class="flex-1 overflow-auto">
+                <div class="flex-1 overflow-auto min-h-[125px]">
```

Comment:
The codemirror base style sheet already sets a height:

```
.CodeMirror {
  /* Set height, width, borders, and global font properties here */
  font-family: monospace;
  height: 300px;
  color: black;
  direction: ltr;
}
```
and the nifi codemirror theme already adds a min-height: 108px;

https://github.com/apache/nifi/blob/main/nifi-frontend/src/main/frontend/libs/shared/src/assets/styles/_codemirror-theme.scss#L65